### PR TITLE
ENH add piff data for y6

### PR DESCRIPTION
**Pull request by beckermr**
cc @esheldon

### CODE REVIEW

**Comment by beckermr** [comment]
@esheldon This one is ready to merge!

File: desmeds/coaddsrc.py [hunk 1]
```diff
@@ -85,9 +85,37 @@ def _do_query(self):
                 'pfw_attempt_id':pai,
                 'magzp': magzp,
             }
-
             info_list.append(info)
 
+        if 'Y6' in self['campaign']:
```
Comment (beckermr) [inline desmeds/coaddsrc.py:90]:
```suggestion
        if 'Y5' in self['campaign'] or 'Y6' in self['campaign']:
```

**Comment by esheldon** [comment]
Do I read correctly that a separate query is used for the piff files?

**Comment by beckermr** [comment]
Yes. In many cases, the piff files cannot be extracted by simply munging the paths of the image. You need to actually pull them out of the database. IDK why this. Robert can likely explain.

**Comment by esheldon** [comment]
I'm happy to merge this if it is working for you.  We can always improve the DB queries if n eeded later.

**Pull request merged**